src/lib/resource/sections/applicant-applied-for-other-compensation-briefly-explain-why-not.js

```diff
@@ -5,22 +5,20 @@ module.exports = {
         l10n: {
             vars: {
                 lng: 'en',
-                context: {
-                    $data:
-                        '/answers/p-applicant-who-are-you-applying-for/q-applicant-who-are-you-applying-for'
-                },
                 ns: 'p-applicant-applied-for-other-compensation-briefly-explain-why-not'
             },
             translations: [
                 {
                     language: 'en',
                     namespace: 'p-applicant-applied-for-other-compensation-briefly-explain-why-not',
                     resources: {
-                        'q-applicant-applied-for-other-compensation-briefly-explain-why-not': {
-                            title:
-                                'Tell us why you have not applied for or received any other compensation',
-                            'title_someone-else':
-                                'Tell us why no application has been made or payment received for any other form of compensation'
+                        'applicant-applied-for-other-compensation-briefly-explain-why-not': {
+                            title: {
+                                myself:
+                                    'Tell us why you have not applied for or received any other compensation or damages',
```

Comment:
No punctuation

Reply:
Question marks are being left of sentences to make them less investigative.